Duckling/Quantity/EN/Corpus.hs

```diff
@@ -72,12 +74,25 @@ allExamples = concat
              , "0.75 cup"
              , ".75 cups"
              ]
+  , examples (simple Piece 3 (Just "bags"))
+             [ "3 Pieces of bags"
+             , "3 piece of BaGs"
+             ]
+  , examples (simple Piece 1 Nothing)
+             [ "1 piece"
+             , "a Piece"
+             ]
```

Comment:
Maybe a test for the abbreviation of Pieces: `pcs`

Reply:
also `pc`

Reply:
That's a good idea to add `pcs`. I will add it. However I am not sure about the `pc`. Is it common to use it? I want to be cautious not to cause false positives.